api/specs/webserver/openapi-computations.yaml

```diff
@@ -1,9 +1,27 @@
 paths:
-  /computation/pipeline/{project_id}/start:
+  /computations/{project_id}:
```

Comment:
WARNING: I am afraid this might have an impact on the meta-modeling (GUI -> backend interaction). Did you change this call in the front-end???

Reply:
of course

Reply:
And that is why I'm asking in the next comment. and yes I changed the call in the frontend, else the e2e tests would not pass.

Reply:
discussed together and fix will be done